specification/network/resource-manager/Microsoft.Network/stable/2018-11-01/publicIpAddress.json

```diff
@@ -3,7 +3,7 @@
   "info": {
     "title": "NetworkManagementClient",
     "description": "The Microsoft Azure Network management API provides a RESTful set of web services that interact with Microsoft Azure Networks service to manage your network resources. The API has entities that capture the relationship between an end user and the Microsoft Azure Networks service.",
-    "version": "2018-11-01"
+    "version": "2018-12-01"
```

Comment:
Why the version doesn't match a folder name?

Reply:
fixed.